### DISCUSSION TITLE
Shutting down System 7.5.3 hangs on desktop (OSHW derivative HW)

### DISCUSSION THREAD

**Discussion by that-ben** [ZuluSCSI/ZuluSCSI-firmware#454] [Q&A]
Hi! Thanks for making this.

Issue: When I shut down my LC III, at the time where the Mac is supposed to go black with a white box saying that it's safe to turn off the Macintosh, instead, ZuluSCSI activity LED flashes once and then it just stays on the desktop.  The mouse cursor moves but I cannot click on anything and the computer is seemingly frozen.  I tried 3 times and it always does that. Is this a firmware bug? It does not do that when booting from a real floppy disk.

I'm on the latest firmware: ZuluSCSI_Pico_DaynaPORT_2024-07-16.bin

**Comment by aperezbios** [comment]
@that-ben Thanks for the report. Given that so far, you're the only person who has reported an issue like this, it seems unlikely to be a firmware issue, but it's always possible. With the information provided, there's not (yet) enough data to be able to determine if it's a hardware problem. There's no mention of the model of the machine you're using your ZuluSCSI with. What is it, and what's the origin of the ZuluSCSI board you're using?

I've converted this issue to a discussion. If it turns out to be an actual bug, we will create an issue for it.

**Comment by that-ben** [comment]
Hi! Thanks.  It's a GBSCSI2 board.  Is that an issue? Also, I believe I wrote "LC III" in my original message, is the Mac I'm using ZuluSCSI with.  If I'm the only one who has reported this issue, it might just be the System file that needs to be reinstalled.  Sometimes, those can become corrupted, it happened even back in the 90's on real hard disks.  I will try a full System 7.5.3 reinstall and I'll post again if it does the same thing or not.

**Reply by aperezbios** [reply]
@that-ben it's been nearly a couple of months since your last post. Were you ever able to figure this out?

**Reply by that-ben** [reply]
I am now in the process of recapping it, been waiting for nearly 2 months for parts from AliExpress 🤦 and once I get the parts, I'll recap it and try a fresh System 7.5.3. I doubt it will make a difference but we'll see.

**Comment by Jakeysp** [comment]
For what it's worth, I have no issues with a V1.1 or V1.0 mini on my LC III with System 7.5.3, it displays the correct shut down screen every time. I don't own the Pico DaynaPORT version though.

**Comment by techfixpros** [comment]
This may be an issue with with installed extensions or software. I have the same issue depending on which extensions are active. I used ConflictCatcher and a bunch of reboots to narrow it down to OT and other network related extensions and control panels, until it shutdown and rebooted properly.
DaynaPort Pico W
SE/30
System 7.5.3

**Comment by that-ben** [comment]
In my case, it turns out that it was bad capacitors making it freeze in that particular place in the OS.  It also did other things that I had noticed since last year like not all the physical RAM was reported in "About this Mac" window.  Either way, I have recapped that LC III since then and forgot that this thread existed.  The LC III does not freeze when shutting down (or anywhere else).  So this has nothing to do with ZuluSCSI or GBSCSI.  Thanks for your interest in this thread, but I'm closing it now 🤣
